test/services/orchestration/index.test.js

```diff
@@ -47,6 +60,40 @@ describe('orchestration service', () => {
     assert.ok(orchestration);
   });
 
+  it('orchestrate multiple notifications', ()=> {
+
+    return createNotification('firstUser')
+      .then(() => {
+        return createNotification('secondUser');
+      })
+      .then(()=> {
+        return Notification.find({});
+      })
+      .then(notifications => {
+        assert.equal(notifications.length, 2);
+        return orchestration
+          .orchestrate(notifications);
+      })
+      .then(()=> {
+        return Notification.find({});
+      })
+      .then(notifications=> {
+        assert.equal(notifications.length, 2);
+        assert.equal(notifications[0].state, Constants.NOTIFICATION_STATES.ESCALATING);
+        assert.equal(notifications[1].state, Constants.NOTIFICATION_STATES.ESCALATING);
+      }).then(()=> {
+        return Escalation
+          .find({})
+          .populate('notification')
+      }).then(escalations => {
+        let users = ['firstUser', 'secondUser'];
+        for (var escalation in escalations) {
+          let username = escalations[escalation].notification.user;
+          assert(users.indexOf(username) >= 0, 'unknown username ' + escalations[escalation].user);
```

Comment:
check that both users have one escalation each. With this test both escalations could be linked to the same user.